locals.tf

```diff
@@ -1,5 +1,6 @@
 locals {
   project_name                       = var.project_name
+  project_name_hash                  = format("%.5s", sha1(local.project_name))
```

Comment:
can this be 6. (6 is less likely to collide)

Reply:
Sure